src/Confluent.Kafka/WatermarkOffsets.cs

```diff
@@ -33,7 +33,7 @@ public class WatermarkOffsets
         ///     The offset of the earlist message in the topic/partition.
         /// </param>
         /// <param name="high">
-        ///     The offset of the last committed message in the topic/partition.
+        ///     The offset of the last stored message in the topic/partition.
```

Comment:
committed is actually more accurate since store sometimes refers to the local in-memory offset-to-commit-store